client/src/app/(onboarding)/signup/page.tsx

```diff
@@ -1,3 +1,208 @@
-export default function Page() {
-  return <h1>Signup placeholder</h1>;
-}
+"use client";
+import { Button, TextField, Typography, Link, Divider } from "@mui/material";
+import "./signup.css";
+import { useState } from "react";
+import "/src/theme/global.css";
+
+export default function SignupPage() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [hasSubmitted, setHasSubmitted] = useState(false); // Tracks form submission
+
+  // Error state for form validation
+  const [error, setError] = useState({
+    email: "",
+    password: "",
+    confirmPassword: ""
+  });
+
+  // Function to handle form validation
+  const validateForm = () => {
+    const formErrors = { email: "", password: "", confirmPassword: "" };
+    let isValid = true;
+
+    if (!email) {
+      formErrors.email = "Email is required";
+      isValid = false;
+    }
+
+    if (!password) {
+      formErrors.password = "Password is required";
+      isValid = false;
+    }
+
+    if (password !== confirmPassword) {
+      formErrors.confirmPassword = "Passwords do not match";
+      isValid = false;
+    } else if (!confirmPassword) {
+      formErrors.confirmPassword = "Confirm password is required";
+      isValid = false;
+    }
+
+    setError(formErrors);
+    return isValid;
+  };
+
+  const handleSubmit = () => {
+    setHasSubmitted(true); // Set the form as submitted
+    if (validateForm()) {
+      console.log("Email: ", email);
+      console.log("Password", password);
+    } else {
+      console.log("Validation failed");
+    }
+  };
+
+  return (
+    <div className="borderline main-container">
```

Comment:
Remove the borderline on the main container. It's causing a uneven borderline on the top border, and this was not in our styles for the pages to have
<img width="699" alt="Screenshot 2024-10-07 at 5 51 25 PM" src="https://github.com/user-attachments/assets/de4d5213-5b24-4174-a249-4a107178dd41">

Reply:
fixed